README.md

```diff
@@ -183,6 +183,17 @@ resource.getSubEntityByClass('inner'); // Entity with 'inner' class
 resource.getSubEntitiesByClass('inner'); // [ All Entities with 'inner' class ]
 ```
 
+#### `Entity.getXByY'es(Array[String|RegExp key])`
+Returns the resource(s) of type _X_ with matching _Y_ values of multiple `key` (or which matches `RegExp key`). The requested _X_ must contain each of the _Y_ values. If the requested _X_ is singular, then the result is either the matching instance of _X_, or undefined. If the requested _X_ is plural, then the result is either an Array of the matching instances of _X_, or an empty Array.
+
+* `getActionByClasses(classes)` - returns [Action](#action) or undefined
```

Comment:
The `getXByY` includes an entry for each method (and a matching JS example), could you add those?

Alsoooo, if you're feeling exceptionally fun, this could also apply to links and fields, both of which can have classes. And if you're feeling _even more fun than that_, a similar set of methods could exist for `rel` as well (where applicable), since those are arrays as well. But mostly YAGNI on that part unless you're super bored.

Reply:
From a code perspective I *think* I added these methods to every place you can get an `X` by a class, but maybe I missed some? Or are you strictly referring to having entries in here for everything? I'll flesh this out for all the existing methods for sure, let me know on the code front if you feel I've missed something (I like the suggestion on `rel` methods as well but I think that's best suited in a follow-up perhaps).

Reply:
Whoops, totally missed this comment... but you implemented what I was trying to get at, so all good :)